packages/storybook/stories/table.stories.tsx

```diff
@@ -496,18 +516,30 @@ export const SortableRows: Story = () => {
             accessorKey: 'column3',
             sortable: true,
         },
+        {
+            header: 'Column 4',
+            accessorKey: 'column4',
+            sortable: true,
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            sortingFn: (rowA: any, rowB: any, columnId: string) => compareEmojiNumber(
```

Comment:
C'est pas des "any". Il te fourni deja les bon types, fait juste enlever tes overrides de types

```suggestion
            sortingFn: (rowA, rowB, columnId) => compareEmojiNumber(
```

```
(property) sortingFn: (rowA: Row<SortableData>, rowB: Row<SortableData>, columnId: string) => number
```